test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/main.py

```diff
@@ -122,9 +122,9 @@ def setup_collection_loader():
 LOOSE_ANSIBLE_VERSION = LooseVersion('.'.join(ansible_version.split('.')[:3]))
 
 
-PLUGINS_WITH_RETURN_VALUES = ('module', )
-PLUGINS_WITH_EXAMPLES = ('module', )
-PLUGINS_WITH_YAML_EXAMPLES = ('module', )
+PLUGINS_REQUIRE_RETURN_VALUES = ('module', 'lookup', 'test', 'filter')
+PLUGINS_REQUIRE_EXAMPLES = ('module', 'inventory', 'lookup', 'filter', 'test')
```

Comment:
Adding new plugin types here should be mentioned in the changelog. This is a user-facing change (where users are core and collection developers).

(Related issue: #71793)

Reply:
done

Reply:
though tempted to remove tests and just auto add '`boolean` return